app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(cp: str) -> None:
+    list_input = cp.split()
+    command, file_in, file_out = list_input
+    if command == "cp" and len(list_input) == 3:
```

Comment:
The check for `command == 'cp'` should be done after verifying that `list_input` has exactly three elements to avoid potential errors.